libraries/botbuilder-repo-utils/src/run.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export type Success = {
+    _type: 'success';
+    statusCode: number;
+};
+
+export function isSuccess(result: Result): result is Success {
+    return result._type === 'success';
+}
+
+export function success(statusCode = 0): Success {
+    return { _type: 'success', statusCode };
+}
+
+export type Failure = {
+    _type: 'failure';
+    statusCode: number;
+    message?: string;
+};
+
+export function failure(statusCode = 1, message?: string): Failure {
```

Comment:
Usage of `failure()` point towards message always being provided (even if it may not be a string). Should `message` be required and the order of the args be flipped?

Reply:
Good call, I'll fix that.